index.d.tmpl

```diff
@@ -7,11 +7,9 @@ declare function depcheck<T>(
 ): Promise<T>;
 
 declare namespace depcheck {
-  interface Node {
-    [key: string]: any;
-  }
+  type Node = Record<string, any>;
 
-  type Parser = (content: string, filePath: string, deps: ReadonlyArray<string>, rootDir: string) => Node;
+  type Parser = (filePath: string, deps: ReadonlyArray<string>, rootDir: string) => Node | string[];
```

Comment:
Not sure I understand this change, the parser doesn't have the `content` parameter? Seems weird to change the type definition and not the code at the same time

Reply:
Oh! Ok 😅